cwms-data-api/src/main/java/cwms/cda/api/watersupply/WaterUserDeleteController.java

```diff
@@ -80,8 +80,7 @@ public void handle(@NotNull Context ctx) {
             DSLContext dsl = getDslContext(ctx);
             String office = ctx.pathParam(OFFICE);
             String locationId = ctx.pathParam(PROJECT_ID);
-            DeleteMethod deleteMode = getDeleteMethod(ctx.queryParamAsClass(METHOD, String.class)
-                    .getOrDefault(DeleteMethod.DELETE_ALL.toString()));
+            DeleteMethod deleteMode = getDeleteMethod(ctx.queryParam(METHOD));
```

Comment:
Don't forget to update the DAO to allow for null

Reply:
Updated DAO